.github/workflows/validate_terraform.yml

```diff
@@ -0,0 +1,103 @@
+name: Terraform
+
+on:
+  pull_request:
+    branches:
+      - master
+      - production
+
+jobs:
+  pre_job:
+    name: Pre Job
+    runs-on: ubuntu-latest
+    outputs:
+      has_operations_change: ${{ steps.skip_check.outputs.operations }}
+      is_fork: ${{ steps.fork_check.outputs.is_fork }}
```

Comment:
Remove this check. We want the build to fail on a fork if anyone makes changes to any of these files. The Action will not normally run unless these files are modified, so this should not affect our normal application PR workflow. This will serve as a precaution and an extra check to ensure outside contributors are not making changes to the infrastructure.

Reply:
Done in next push.